src/cli.rs

```diff
@@ -0,0 +1,14 @@
+use crate::config::config_path;
+use clap::Parser;
+
+#[derive(Parser, Debug)]
+#[clap(version, about, long_about = None)]
+pub struct CliArgs {
+    #[clap(short, long,value_parser, default_value_t = config_path().unwrap().to_str().unwrap().to_string())]
+    pub config: String,
+}
```

Comment:
```suggestion
/// Docstring...
const DEFAULT_CONFIG_PATH: &str = config_path().expect("Could not get default config path").to_str().expect("could not convert to &str");

/// Docstring...
#[derive(Parser, Debug)]
#[clap(version, about, long_about = None)]
pub struct CliArgs<'a> { // Maybe you can do this. Otherwise convert to String.
    /// Docstring...
    #[clap(short, long,value_parser, default_value_t = DEFAULT_CONFIG_PATH)]
    pub config: &'a str,
}
```